parlai/tasks/integration_tests/agents.py

```diff
@@ -30,6 +30,120 @@
 NUM_TEST = 100
 
 
+class FixedDialogCandidateTeacher(FixedDialogTeacher):
+    """
+    Base Candidate Teacher.
+
+    Useful if you'd like to test the FixedDialogTeacher
+    """
+
+    def __init__(
+        self,
+        opt,
+        shared=None,
+        vocab_size=VOCAB_SIZE,
```

Comment:
would you be interested in starting to do type hints?

Reply:
Sure